components/ProductInfo/middle/ShirtOptions.js

```diff
@@ -92,22 +119,29 @@ const ShirtOptions = () => {
             </span>
           </div>
           <ul className={style.colorOptions}>
-            {shirtColors.map((shirt, i) => {
+            {shirtColors.map(({ color, img, displayImg }, i) => {
               return (
                 <li className={style.option} key={i}>
-                  <div className={style.optionBox}>
-                    <button
-                      className={style.optionBtn}
-                      onMouseEnter={() => mouseEnter(shirt.img)}
-                      onMouseLeave={() => mouseLeave()}
-                    >
-                      <img
-                        className={style.optionIMG}
-                        alt={shirt.color}
-                        src={shirt.img}
-                      ></img>
-                    </button>
-                  </div>
+                  {/* <div className={style.optionBox}> */}
+                  <button
+                    type="submit"
+                    className={`${
+                      imageDisplayBtnArr[0] === displayImg && style.active
+                    } ${style.optionBtn}`}
+                    onMouseEnter={() => mouseEnter(displayImg)}
+                    onMouseLeave={() => mouseLeave()}
+                    onClick={(e) => {
+                      e.preventDefault();
+                      mouseClick(displayImg);
+                    }}
+                  >
+                    <img
+                      className={style.optionIMG}
+                      alt={color}
+                      src={img}
+                    ></img>
+                  </button>
+                  {/* </div> */}
```

Comment:
refactored your map to deconstruct the shirt obj, allowing us to reference props without the need for shirt.[prop]
` <button
                    type="submit"
                    className={`${
                      imageDisplayBtnArr[0] === displayImg && style.active
                    } ${style.optionBtn}`}
                    onMouseEnter={() => mouseEnter(displayImg)}
                    onMouseLeave={() => mouseLeave()}
                    onClick={(e) => {
                      e.preventDefault();
                      mouseClick(displayImg);
                    }}
                  >`
                  
                  had to change button to type submit because on every click it reloaded the page for whatever reason, so i made it type submit and prevented the default to not reload the page. 
                  
line 129 if the firstbtn img url found on the left side of the imagedisplay is the same as the displayImg of the current shirtColor btn set it as active.